src/core.jl

```diff
@@ -19,3 +19,187 @@ immutable ImageEdge
     index2::Int
     weight::Float64
 end
+
+"""
+    G, vert_map = region_adjacency_graph(seg, weight_fn)
+
+Constructs a region adjacency graph from the `SegmentedImage`. It returns the RAG
```

Comment:
Maybe put `(RAG)` after "region adjacency graph" since you use that abbreviation later.